setup.cfg

```diff
@@ -1,4 +1,5 @@
 [flake8]
+max-line-length: 120
```

Comment:
Hmm. I don't recall that discussion, and have never been in favor of 80 column limits, so it seems unlikely I was there. However I don't wish to re-legislate if it's already been decided. I'll change this one.